### DISCUSSION TITLE
[PSD7] Exercise 2.17

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#158] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210018738-5bcc05f5-25d5-48bc-b1e2-580eaabe1c8b.png)

**Comment by Elia-Belli** [comment]
- (a) $Y = B (C + \bar C) + \bar A \bar B \bar C = (\bar A + B)(B +\bar C) = \bar A B + \bar A \bar C + B + B \bar C = B + \bar A \bar C$
- (b) $Y = (\bar A (\overline {\bar A B})(\overline {\bar A \bar B})) + \bar A B = (\bar A (A + \bar B) (A+B)) + \bar A B = (\bar A (A+ A \bar B + \bar B A)) + \bar A B = \bar A A + \bar A B = \bar A B$
- (c) $AB (C+D+E) + AC (D+E) + \bar B \bar C (D+E) + \bar D \bar E (\bar A + \bar B + \bar C) =$
$AB (C +D+E) +(D+E)(AC + \bar B \bar C)+ \bar D \bar E (\bar A + \bar B + \bar C)$ 
Oltre De Morgan non è possibile semplificare altro, ho riscritto la funzione raccogliendo alcuni membri per realizzare un circuito leggermente più piccolo.
![circuit](https://user-images.githubusercontent.com/107944476/211662026-334861f9-e49b-4c97-a55a-3920505344b9.jpg)
- **(c) alternativa**: ho realizzato la K-map e la POS minima, che contenendo meno termini permette di realizzare un circuito più semplice (che lascio realizzare alla vostra immaginazione).
![K](https://user-images.githubusercontent.com/107944476/211663758-64d4fc5a-7f37-47af-a045-f0587269cb72.png)
$Y = (E + A + \bar B)(E + A +\bar C)(A + \bar B + \bar D)(A +\bar C +\bar D)(\bar E +\bar A +B + \bar C + \bar D)$

**Reply by neoSnakex34** [reply]
il c è errato

**Reply by neoSnakex34** [reply]
![image](https://github.com/user-attachments/assets/e952ed6c-0c99-42cb-b0c1-6d964e398ff6)

**Comment by Emanuele-Dambrosio** [comment]
A)https://www.boolean-algebra.com/?q=QkMre0FCQ30rYntjfQ==
B)ok
C)https://www.boolean-algebra.com/?q=

**Comment by fraxb04** [comment]
mi vengono meno porte nell'esercizio c
![image](https://user-images.githubusercontent.com/122573871/212700037-2f5ae4b2-d1cc-4b2f-953e-a6f1d763b6dc.png)

**Comment by FeddyLix17** [comment]
tutte le semplificazioni di @Elia-Belli sono corrette, i circuiti (a) e (b) mi vengono analoghi ai suoi, il (c) molto simile lo carico qui di seguito
![circuito PSD7- Esercizi Exercise 2 17 (c)](https://user-images.githubusercontent.com/66442720/213144679-998735d0-79ab-441d-a67a-f12abbd04fdf.jpg)

**Comment by niccolo-fato** [comment]
![photo_2024-11-17_15-36-14](https://github.com/user-attachments/assets/8bb43755-7af7-4ef2-8ed3-7134e74f0796)
![photo_2024-11-17_15-36-17](https://github.com/user-attachments/assets/091f7628-376b-496b-8a93-7e28d10f6b3d)
